src/include/daos_api.h

```diff
@@ -33,6 +33,16 @@
 extern "C" {
 #endif
 
+enum {
```

Comment:
Does "enum" contradict with "uint64_t flags"? That is, is it possible to define flags using bit 32 and above using "enum"?

Reply:
Hm, expressing 64 bits flags via enum may be cause trouble (also depend on compiler), it is general issue for DAOS now. I prefer to convert them to "#define" when the flags value exceed 32-bits in the future.